IBM/carbon-components-react
# fix(code-snippet): add screen reader accessibility improvements

ref #1953 

### change log

- adds a role of alert to the copy verification popup on the code snippet making it accessible to screen readers.

  _this gets the react code snippet in line with changes made to the vanilla component in [this pull request](https://github.com/IBM/carbon-components/pull/1969) 🏄_

- adds more explicit inline code snippet example to storybook
  _this new `aria-label` demonstrates intended use and calls out to user the buttons function_

- removes now unused `copyLabel` prop

- adds `aria-describedby` and corresponding `id` so the code snippet is read aloud

### to test
Run a screen reader. Tab to the code snippet and hit space press space to copy. The copy verification should be read aloud.
